src/arch/x86/kvm/x86_cpu.cc

```diff
@@ -953,6 +976,10 @@ X86KvmCPU::updateKvmStateFPUXSave()
     xsave_hdr.xstate_bv.fpu = 1;
     xsave_hdr.xstate_bv.sse = 1;
 
+    // enable avx bitmap and run the AVX state update function
```

Comment:
can you please update your comment here to explain why this is needed? / what it is needed for?